src/Analyzers/Core/Analyzers/RemoveUnnecessaryImports/AbstractRemoveUnnecessaryImportsDiagnosticAnalyzer.cs

```diff
@@ -117,6 +118,7 @@ public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics
             }
         }
 
+        public CodeActionRequestPriority RequestPriority => CodeActionRequestPriority.Normal;
```

Comment:
Effectively, everything that previously produced a 'high pri code action' is in the 'high pri request' category.  Anything that produces a normal/low pri code action is in the normal request category.  I didn't want to change anything in that regard here.  Right now we've had a long historical precedent of recognizing two code actions (one analzyer, one refactoring) that are so important that they should beat out everything else.  This PR codifies things so that not only do those come first, they are computed first so we can display them asap for the user.